Integrations/SplunkPy/SplunkPy.py

```diff
@@ -24,6 +24,10 @@
 VERIFY_CERTIFICATE = not bool(demisto.params().get('unsecure'))
 FETCH_LIMIT = int(demisto.params().get('fetch_limit', 50))
 FETCH_LIMIT = max(min(200, FETCH_LIMIT), 1)
+FETCH_TIME = demisto.params().get('fetch_time')
+TIME_IN_MINUTES_DICT = {'minute': 1, 'minutes': 1, 'hour': 60, 'hours': 60, 'day': 24 * 60, 'days': 24 * 60,
```

Comment:
parse_date_range should receive the relevant format and return a datetime object.
You can then parse that datetime to minutes.
What am I missing?

Reply:
lets talk about it over zoom and ill explain why